src/ui/qgsoptionsbase.ui

```diff
@@ -332,7 +332,7 @@
        <item>
         <widget class="QStackedWidget" name="mOptionsStackedWidget">
          <property name="currentIndex">
-          <number>0</number>
+          <number>6</number>
```

Comment:
Would this change the default options view?

Reply:
The default page is stored in the settings, so it shouldn't change, but we can keep it.

```suggestion
          <number>0</number>
```